src/support/registerSupport.ts

```diff
@@ -24,4 +25,10 @@ export const registerSupport = (): void => {
     Cypress.Commands.overwrite('fixture', fixture);
 
     Cypress.Commands.add('step', step);
+
+    // Since registerSupport() function is called in each repo from e2e.js,
+    // attaching the JavaScript errors logger hooks here ensures that logger is initialized automatically
+    // for all tests without needing to call it explicitly in each test file.
+    // This is useful for capturing and logging JavaScript errors across all tests.
+    jsErrorsLogger.attachHooks();
```

Comment:
Does it mean that now every project will fail fast on the first error/warning js error ?

Reply:
That could be difficult for some project to handle all warnings and everythings that may be involved, and dev would probably require upgrade to the latest for other reasons.
This will have probably to be discuss with all the devs first, to find the good balance.
At least there is ways to disable it, so it's a good thing for project with complex cases with warnings/errors.
I would be in favor of bumping the major version of the project due to this new behavior for peoples that's gonna upgrade to it.